pythia/cleaning/tests/test_midnight_rotation.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+from pathlib import Path
+
+import pytest
+from pythia.cleaning import MidnightRotation
+from pythia.seo import Sunspotter
+
+path = Path(__file__).resolve().parent.parent.parent.parent / "data/all_clear"
+
+
+@pytest.fixture
+def midnight_rotation():
+    return MidnightRotation()
+
+
+@pytest.fixture
+def sunspotter(properties_columns, timesfits_columns):
+    return Sunspotter(timesfits=path / "lookup_timesfits.csv",
+                      properties=path / "lookup_properties.csv",
+                      delimiter=';',
+                      timesfits_columns=timesfits_columns,
+                      properties_columns=properties_columns)
+
+
+@pytest.fixture
+def obsdate():
+    return '2000-01-01 12:47:02'
```

Comment:
where are you using this fixture?

Reply:
Ah, I forgot this. I was using this before I parameterized stuff.